deepseg.cc

```diff
@@ -478,16 +479,15 @@ int main(int argc, char* argv[]) {
 		// wait for next frame
 		while (capinfo.cnt == oldcnt) usleep(10000);
 		oldcnt = capinfo.cnt;
-		int e1 = cv::getTickCount();
 		ti.waitns=timestamp();
 
 		// switch buffer pointers in capture thread
-		pthread_mutex_lock(&capinfo.lock);
+		capinfo.lock.lock();
 		ti.lockns=timestamp();
 		cv::Mat *tmat = capinfo.grab;
 		capinfo.grab = capinfo.raw;
 		capinfo.raw = tmat;
-		pthread_mutex_unlock(&capinfo.lock);
+		capinfo.lock.unlock();
```

Comment:
dito, use braces to denote region for lock to be held.

Reply:
done